docs/xls-70d-credentials/reference/credentialids-field.md

```diff
@@ -0,0 +1,46 @@
+# CredentialIDs Field
+
+The `CredentialIDs` field is a new field added to the following transaction types by XLS-70 (Credentials):
+
+- [AccountDelete][]
+- [EscrowFinish][]
+- [Payment][]
+- [PaymentChannelClaim][]
+
+This field's definition is as follows:
+
+| Field           | JSON Type        | [Internal Type][] | Required? | Description |
+|:----------------|:-----------------|:------------------|:----------|:------------|
+| `CredentialIDs` | Array of Strings | Vector256 | No | Set of Credentials to authorize a deposit made by this transaction. Each member of the array must be the ledger entry ID of a Credential entry in the ledger and the sender of this transaction must be the subject of the credential. |
+
+You can send money to an account that uses [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth) by providing this field with an exact set of credentials that are preauthorized by the recipient; the set of credentials must match a [DepositPreauth entry](./depositpreauth-entry.md) in the ledger.
```

Comment:
...by the recipient. The set of credentials...

[The paragraph consists of only the one sentence, so it can easily be broken in two.]